packages/core/src/routes/session/passwordless.ts

```diff
@@ -102,7 +108,7 @@ export default function passwordlessRoutes<T extends AnonymousRouter>(
 
       await verifyPasscode(jti, flow, code, { phone });
 
-      await provider.interactionResult(ctx.req, ctx.res, {
+      await assignVerificationResult(ctx, provider, {
```

Comment:
we can move other params to this function:

```
await assignVerificationResult(ctx, provider, flow);
```

Reply:
Updated input parameters.